### PR TITLE
pep8ify a couple of variable names.

### PR DESCRIPTION
**Pull request by anntzer**
The deleted block in backend_pdf
(`if (width > paperWidth or height > paperHeight) and isEPSF:`)
can never be triggered as it's in the `else` clause of an `if isEPSF`
just above.  Looks like this came in in 946c5f2 (2009) and no one ever
complained about the missing log, so I think it can just be deleted.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/backends/backend_ps.py [hunk 1]
```diff
@@ -892,7 +892,7 @@ def _print_figure(
         metadata must be a dictionary. Currently, only the value for
         the key 'Creator' is used.
         """
-        isEPSF = format == 'eps'
+        is_epsf = format == 'eps'
```
Comment (timhoffm) [inline lib/matplotlib/backends/backend_ps.py]: Whatever the 'f' stands for...

Comment (anntzer) [reply lib/matplotlib/backends/backend_ps.py]: looks like just an alternate extension for eps... https://en.wikipedia.org/wiki/Encapsulated_PostScript

Comment (timhoffm) [reply lib/matplotlib/backends/backend_ps.py]: Would change to is_eps then. It‘s equal, more common, and the string we use for the format.

Comment (anntzer) [reply lib/matplotlib/backends/backend_ps.py]: done

**Pull request merged**